client/js/spotify-gdpr.js

```diff
@@ -0,0 +1,203 @@
+// @ts-check
+/* global c3 */
+
+// most recent ordering of top songs
+const latest = [
+  'song3',
```

Comment:
Why are songs 4, 6, 8, and 9 missing from this list. Also can you add a clarifying comment why only songs 0-9 are out of order and the rest are linear?

Reply:
Follow up, it's looking like this file is a bunch of static mock data. If that's the case, it would be helpful to have a file level comment explaining this.